source/Paths.hx

```diff
@@ -355,7 +355,7 @@ class Paths
 			localTrackedAssets.push(path);
 			return currentTrackedAssets.get(path);
 		}
-		trace('oh no its returning null NOOOO');
+		trace('oh no ' + key + ' is returning null NOOOO');
```

Comment:
numbers can't explain how ingenious this single addition is, but words can.

approved/10